task/runner.go

```diff
@@ -258,4 +261,31 @@ func errorInfo(err error) *data.ErrorInfo {
 	return &data.ErrorInfo{Message: err.Error(), Unretriable: IsUnretriable(err)}
 }
 
+func humanizeError(err error) error {
+	if err == nil {
+		return nil
+	}
+
+	errMsg := strings.ToLower(err.Error())
+
+	if strings.Contains(errMsg, "unexpected eof") {
+		return errors.New("file download failed")
+	} else if strings.Contains(errMsg, "multipartUpload: upload multipart failed") {
```

Comment:
got an uppercase here
```suggestion
	} else if strings.Contains(errMsg, "multipartupload: upload multipart failed") {
```

Reply:
Nice catch!